server/channels/api4/channel.go

```diff
@@ -806,6 +807,13 @@ func getPinnedPosts(c *Context, w http.ResponseWriter, r *http.Request) {
 	if err := clientPostList.EncodeJSON(w); err != nil {
 		c.Logger.Warn("Error while writing response", mlog.Err(err))
 	}
+
+	if !isMember {
+		auditRec := c.MakeAuditRecord("viewed_pinned_posts_without_membership", audit.Success)
+		defer c.LogAuditRecWithLevel(auditRec, app.LevelContent)
```

Comment:
According [to our docs](https://docs.mattermost.com/manage/logging.html#log-levels), the `audit-content` level is used for

> Content changes. This log level can generate considerably more records than the other audit log levels.

Should we introduce a new level that covers this specific use case? Reading a post doesn't quite fit the current definition, does it?

Reply:
The search post endpoint uses `audit-api` (https://github.com/mattermost/mattermost/blob/2a35a97a18773e69772db9e3d2d1d9c5e7fbe011/server/channels/api4/post.go#L821-L823). Should we do the same here?